package.json

```diff
@@ -161,10 +161,12 @@
     "typescript": "4.8.3"
   },
   "resolutions": {
+    "@ledgerhq/hw-transport": "^6.27.4",
     "@typescript-eslint/eslint-plugin": "5.38.1",
     "eslint-plugin-react": "7.31.8"
   },
   "alias": {
+    "@ledgerhq/devices": "@ledgerhq/devices/lib-es",
```

Comment:
Can you add a comment like `"//": "parcel doesn't support subpath exports #issue"`